### PR TITLE
Transfer Project to Another Team for appwrite 1.4.x

### PR DESCRIPTION
**Pull request by lohanidamodar**
## What does this PR do?

- Transfer Project to another team

https://github.com/appwrite/console/assets/6360216/3b6172ad-0a87-47fa-8b76-21800df27cf2

## Test Plan

(Write your test plan here. If you changed any code, please provide us with clear instructions on how you verified your changes work.)

## Related PRs and Issues

- https://github.com/appwrite/appwrite/pull/5483

### Have you read the [Contributing Guidelines on issues](https://github.com/appwrite/appwrite/blob/master/CONTRIBUTING.md)?

✅

### CODE REVIEW

File: src/routes/console/project-[project]/settings/+page.svelte [hunk 1]
```diff
@@ -14,14 +15,19 @@
     import { base } from '$app/paths';
     import { page } from '$app/stores';
     import { Submit, trackEvent, trackError } from '$lib/actions/analytics';
+    import InputSelect from '$lib/elements/forms/inputSelect.svelte';
```
Comment (TorstenDittmann) [inline src/routes/console/project-[project]/settings/+page.svelte:18]: Let's use the named import 👍🏻

File: src/routes/console/project-[project]/settings/transferProject.svelte [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+                type: 'error',
+                message: error.message
+            });
+            trackError(error, Submit.ProjectUpdateTeam);
+        }
+    };
+</script>
+
+<Modal bind:show onSubmit={handleTransfer} icon="exclamation" state="warning" headerDivider={false}>
+    <svelte:fragment slot="header">Delete Project</svelte:fragment>
```
Comment (TorstenDittmann) [inline src/routes/console/project-[project]/settings/transferProject.svelte:48]: title :-D

Comment (lohanidamodar) [reply src/routes/console/project-[project]/settings/transferProject.svelte]: fixed

File: src/routes/console/project-[project]/settings/+page.svelte (hunk 1 above)
Comment (lohanidamodar) [reply src/routes/console/project-[project]/settings/+page.svelte]: fixed

**Comment by chenparnasa** [comment]
After the tweaks we did in the design review, looks good from my side 👍

**Review by TorstenDittmann — APPROVED** [review]

**Pull request merged**